images/01-base/Dockerfile

```diff
@@ -39,7 +39,8 @@ RUN rm /sbin/poweroff /sbin/reboot /sbin/halt && \
     ln -s /usr/share/dhcpcd/hooks/10-wpa_supplicant /lib/dhcpcd/dhcpcd-hooks/ && \
     rm -f /usr/share/bash-completion/completions/* && \
     sed -i 1,10d /etc/rsyslog.conf && \
-    echo "*.*                /var/log/syslog" >> /etc/rsyslog.conf
+    echo "*.*                /var/log/syslog" >> /etc/rsyslog.conf && \
+    chmod +x /usr/bin/subscriber
```

Comment:
better to use `os-subscriber`

Reply:
And we don't need `chmod` if we pre-set permission for this file.

Reply:
Already modified